pyproject.toml

```
@@ -20,3 +20,80 @@ multi_line_output = 3
 include_trailing_comma = "True"
 force_grid_wrap = 0
 use_parentheses = "True"
+
+[project]
+name = "mozregression"
+dynamic = ["version"]
+description = "Regression range finder for Mozilla nightly builds"
+readme = "README.md"
+dependencies = [
+  "glean_sdk>=60.3.0",
+  "beautifulsoup4>=4.7.1",
+  "colorama>=0.4.1",
+  "configobj>=5.0.6",
+  "distro>=1.8.0",
+  "importlib_resources>= 5.10",
+  "mozdevice>=4.1.0,<5",
+  "mozfile>=2.0.0",
+  "mozinfo>=1.1.0",
+  "mozinstall>=2.0.0",
+  "mozlog>=4.0",
+  "mozprocess>=1.3.1",
+  "mozprofile>=2.2.0",
+  "mozrunner>=8.0.2",
+  "mozversion>=2.1.0",
+  "redo>=2.0.2",
+  "requests>=2.21.0",
+  "taskcluster>=6.0.0",
+]
+requires-python = ">= 3.8.1"
+license = {text = "MPL 2.0"}
+authors = [
+  { name = "Mozilla Automation and Tools Team", email = "tools@lists.mozilla.org" }
```

Making it more consistent with the other similar entries.

```suggestion
  {name = "Mozilla Automation and Tools Team", email = "tools@lists.mozilla.org"}
```